Plugins/lib/windows/arm64/gilzoide-sqlite-net.dll.meta

```diff
@@ -48,25 +50,44 @@ PluginImporter:
     second:
       enabled: 1
       settings:
-        CPU: AnyCPU
+        CPU: None
   - first:
       Standalone: OSXUniversal
     second:
       enabled: 1
       settings:
-        CPU: AnyCPU
+        CPU: None
   - first:
       Standalone: Win
     second:
-      enabled: 0
+      enabled: 1
       settings:
-        CPU: AnyCPU
+        CPU: None
   - first:
       Standalone: Win64
+    second:
+      enabled: 1
+      settings:
+        CPU: None
```

Comment:
I don't think this should be disabled. Even if it requires Unity 6, why not keep this configured so that people can try it out? I mean, if it doesn't work the way it is, we can fix later, but until we know that this doesn't work, just let it be. What do you think?